tests/unit/fem/test_2d_bilininteg.cpp

```diff
@@ -2090,7 +2090,7 @@ TEST_CASE("2D Bilinear Scalar Weak Curl Integrators",
           "[NonlinearFormIntegrator]")
 {
    int order = 2, n = 1, dim = 2;
-   double cg_rtol = 1e-14;
+   // double cg_rtol = 1e-14; //declared but never referenced
```

Comment:
I think these are fine to remove.

Reply:
Agreed.  No compelling reason to leave them in and they pollute our compiler warnings.

Reply:
Done